test/integration/azureService.js

```diff
@@ -5,7 +5,7 @@ const azureService = require('../../app/lib/azureService');
 const expect = chai.expect;
 // writing to a shared area, timestamp file to avoid collisions
 const name = `${new Date().getTime()}test.json`;
-const timeout = 5000;
+const timeout = 15000;
```

Comment:
Does this need increasing to `15000` or did it fail as a one off and could it be left at `5000`?

Reply:
I was getting a lot of failures locally due to poor network speeds in the office during lunch time until I bumped the timeout up. This is with a network cable, not wifi.